Userland/Utilities/netstat.cpp

```diff
@@ -132,6 +136,13 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
         return DeprecatedString::formatted("{}/{}", pid, program.value());
     };
 
+    auto get_formatted_user = [&](i32 uid) {
+        if (uid == -1)
+            return DeprecatedString("-");
+
+        return DeprecatedString::number(uid);
+    };
```

Comment:
Suggestion: make these `String`s instead, and use `.to_deprecated_string()` closer to the source of the deprecated usage :^)

Reply:
I'm really not sure if this variant is any better..

Reply:
This should be good now.